platform/android/src/style/layers/heatmap_layer.cpp

```diff
@@ -79,6 +79,12 @@ namespace android {
         layer.as<mbgl::style::HeatmapLayer>()->HeatmapLayer::setHeatmapIntensityTransition(options);
     }
 
+    jni::Object<jni::ObjectTag> HeatmapLayer::getHeatmapColor(jni::JNIEnv& env) {
+        using namespace mbgl::android::conversion;
+        Result<jni::jobject*> converted = convert<jni::jobject*>(env, layer.as<mbgl::style::HeatmapLayer>()->HeatmapLayer::getHeatmapColor());
+        return jni::Object<jni::ObjectTag>(*converted);
```

Comment:
@tobrun Note that in https://github.com/mapbox/mapbox-gl-native/pull/11074/files#diff-c48bbf0fbc172c9c08815dbfc04e4c66R78 (which I think is the iOS analogue to this), there's a check for whether `layer.as<mbgl::style::HeatmapLayer>()->HeatmapLayer::getHeatmapColor()` is undefined, and if so, it uses `getDefaultHeatmapColor()` instead.

Reply:
thanks for flagging, addressed with https://github.com/mapbox/mapbox-gl-native/pull/11220/commits/e5ce57b0bcde8488fc038b8d841a5d7abdf56f0e